.gitlab-ci.yml

```diff
@@ -358,13 +358,13 @@ gce_centos7-flannel-addons:
   except: ['triggers']
   only: [/^pr-.*$/]
 
-gce_centos-weave-kubeadm:
+gce_centos-calico-kubeadm:
```

Comment:
If you are renaming this job, you need to rename it everywhere. Look at lines 251 and 468

Reply:
done